org.scala-ide.sdt.core/src/org/scalaide/refactoring/internal/OrganizeImports.scala

```diff
@@ -212,7 +215,7 @@ class OrganizeImports extends RefactoringExecutorWithoutWizard {
     }
   }
 
-  class OrganizeImportsScalaIdeRefactoring(file: ScalaSourceFile) extends ScalaIdeRefactoring("Organize Imports", file, 0, 0) {
+  class OrganizeImportsScalaIdeRefactoring(file: ScalaSourceFile) extends ScalaIdeRefactoring(OrganizeImports, "Organize Imports", file, 0, 0) {
```

Comment:
I didn't think of replacing the previous description with the new one that is part of the feature. I'm going to have a look if it is possible to replace them all.